buildSrc/version.properties

```diff
@@ -2,9 +2,10 @@ opensearch        = 3.0.0
 lucene            = 9.8.0
 
 bundled_jdk_vendor = adoptium
-bundled_jdk = 20.0.2+9
+bundled_jdk = 21+35
 # See please https://github.com/adoptium/temurin-build/issues/3371
 bundled_jdk_linux_ppc64le = 20+36
```

Comment:
Still waiting for mac/aarch64 and linux/ppc64le JDK-21 bundles from Eclipse Temurin (see please https://adoptium.net/temurin/releases/)